packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -223,27 +242,37 @@ export class Pipeline extends PipelineBase {
       throw new Error('Only one of artifactBucket and crossRegionReplicationBuckets can be specified!');
     }
 
+
+    // @deprecated(v2): switch to default false
```

Comment:
I don't agree that `true` is a good default for this. We are charging customers even though they don't need this particular feature, and because of the error message it is quick and obvious what you need to do should you want this after all.